openml/runs/functions.py

```diff
@@ -483,8 +503,11 @@ def _calculate_local_measure(sklearn_fn, openml_name):
                     for j, class_label in enumerate(task.class_labels):
```

Comment:
I know this code was there before too. But could you change `class_label` to `_` since the variable is unused?